src/Routes.js

```diff
@@ -2,18 +2,18 @@
 const routes = {
   HomePage: {
     url: '/',
-    component: 'src/pagesComponents/HomePage.js',
+    component: 'src/pages/HomePage.js',
```

Comment:
Maybe I'm missing something, but these paths don't seem to match the actual file paths below?

For example, this page is renamed to `src/pages/index.js`, not `src/pages/HomePage.js`, and the "About" page is `src/pages/about.js`, not `src/pages/AboutPage.js`.

If I'm missing something, feel free to ignore!

Reply:
No you right there is a mismatch